pkg/apis/build/v1alpha1/build_template_defaults.go

```diff
@@ -0,0 +1,6 @@
+package v1alpha1
+
+// SetDefaults for build template
+func (b *BuildTemplate) SetDefaults() {
```

Comment:
Can we just put this into `build_template_types.go` ? This separate file doesn't add much.